### PR TITLE
Add list of FPWD critical issues

### PR DESCRIPTION
**Pull request by wseltzer**
List the issues that, before publication of the FPWD, the WG and CG identified to be considered before CR publication.

***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/623.html" title="Last updated on Aug 6, 2024, 5:49 PM UTC (da3e25a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/623/10794f7...da3e25a.html" title="Last updated on Aug 6, 2024, 5:49 PM UTC (da3e25a)">Diff</a>

### CODE REVIEW

**Comment by wseltzer** [comment]
This list of issues: https://github.com/fedidcg/FedCM/issues?q=is%3Aissue+is%3Aopen+label%3AFPWD+sort%3Acreated-asc

**Review by samuelgoto — APPROVED** [review]
LGTM

**Review by npm1 — APPROVED** [review]

File: spec/index.bs [hunk 1]
```diff
@@ -2937,3 +2937,27 @@ Note: write down the Acknowledgements section.
   }
 }
 </pre>
+
+<!-- ====================================================================== -->
+# FPWD Issues # {#openissues}
+<!-- ====================================================================== -->
+Note: The WG has labeled the following issues as critical open issues that must be considered before publication of a Candidate Recommendation.
```
Comment (martinthomson) [inline spec/index.bs:2944]:
```suggestion
Note: The WG has labeled the following issues as critical open issues that must be resolved before publication of a Candidate Recommendation.
```

**Review by martinthomson — CHANGES_REQUESTED** [review]
I assert that "consideration" is too weak for this.  These need resolutions that are backed by consensus.

**Comment by wseltzer** [comment]
> I assert that "consideration" is too weak for this. These need resolutions that are backed by consensus.

That's consistent with the [W3C Process](https://www.w3.org/policies/process/#transition-reqs) requirement that the group "MUST [formally address](https://www.w3.org/policies/process/#formally-addressed) all issues raised about the document since the previous [maturity stage](https://www.w3.org/policies/process/#maturity-stages)."

**Comment by wseltzer** [comment]
On the 30 July WG call, it was suggested to adopt the language directly from the W3C Process, "formally addressed." PR updated accordingly.

File: spec/index.bs [hunk 2]
```diff
@@ -2937,3 +2937,27 @@ Note: write down the Acknowledgements section.
   }
 }
 </pre>
+
+<!-- ====================================================================== -->
+# FPWD Issues # {#openissues}
+<!-- ====================================================================== -->
+Note: The WG has labeled the following issues as critical open issues that must be formally addressed before publication of a Candidate Recommendation.
```
Comment (samuelgoto) [inline spec/index.bs:2944]: I'm still seeing issues in this [label](https://github.com/fedidcg/FedCM/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc+label%3Achrome) that isn't represented in this [label](https://github.com/fedidcg/FedCM/issues?q=is%3Aissue+is%3Aopen+label%3AFPWD+sort%3Acreated-asc), or in this PR.

Can I assume that these will eventually match up?

Comment (wseltzer) [reply spec/index.bs:2944]: Thanks Sam, https://github.com/fedidcg/FedCM/pull/630 adds these issues to the list.

**Comment by wseltzer** [comment]
[Updated call for consensus](https://lists.w3.org/Archives/Public/public-fedid-wg/2024Aug/0001.html) until 13 August.

File: spec/index.bs (hunk 2 above)
Comment (samuelgoto) [reply spec/index.bs:2944]: LGTM

**Review by samuelgoto — APPROVED** [review]

**Comment by bvandersloot-mozilla** [comment]
> > I assert that "consideration" is too weak for this. These need resolutions that are backed by consensus.
> 
> That's consistent with the [W3C Process](https://www.w3.org/policies/process/#transition-reqs) requirement that the group "MUST [formally address](https://www.w3.org/policies/process/#formally-addressed) all issues raised about the document since the previous [maturity stage](https://www.w3.org/policies/process/#maturity-stages)."

That works as well for Mozilla

**Pull request merged**